students/urtnowski_daniel/lesson_03_functions/calculator.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
```

Comment:
You've got some left-overs here, and the module naming should be lowercase, like this one. Also if you create two files with names only differing in case, this might break things for people with Windows.